themes/custom/essex/css/overrides.css

```diff
@@ -838,4 +847,4 @@ tbody tr {
   height: auto;
   object-fit: cover;
   aspect-ratio: 9/5;
-}
+}
```

Comment:
It would be better not to remove the newline at the end.

Reply:
Thanks, done